src/Microsoft.AspNetCore.Cryptography.KeyDerivation/PBKDF2/DefaultPbkdf2Provider.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if NETCOREAPP2_0 
+// Rfc2898DeriveBytes in .NET Standard 2.0 only supports SHA1
+
+using System;
+using System.Diagnostics;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Microsoft.AspNetCore.Cryptography.KeyDerivation.PBKDF2
+{
+    /// <summary>
+    /// Implements Pbkdf2 using <see cref="Rfc2898DeriveBytes"/>.
+    /// </summary>
+    internal sealed class DefaultPbkdf2Provider : IPbkdf2Provider
```

Comment:
Suggest renaming this type to something else since it's not the default - perhaps _NetCoreAppPbkdf2Provider_?

Reply:
👍 Couldn't think of a good name either. Let's go with NetCoreAppPbkdf2Provider